llvm/test/Transforms/SimpleLoopUnswitch/nontrivial-unswitch-pred-removed.ll

```diff
@@ -0,0 +1,24 @@
+; RUN: opt -passes='loop-mssa(simple-loop-unswitch<nontrivial>),print<memoryssa>' -verify-memoryssa -disable-output -S < %s 2>&1 | FileCheck %s
+; RUN: opt -passes='loop-mssa(simple-loop-unswitch<nontrivial>,licm)' -verify-memoryssa -disable-output -S 
```

Comment:
If possible, I like `/llvm/utils/update_test_checks.py`, but verifying that it does not crash is sufficient.